codeFellowship/codeFellowship/src/main/resources/templates/user-info.html

```diff
@@ -1,23 +1,26 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<link rel="stylesheet" type="text/css" href="/css/styles.css">
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
 </head>
 <body>
 <h3 th:if="${username} != null" th:text="${'Current browsing user: ' + username}"></h3>
-
+<div th:replace="/fragments/error-fragment :: siteuser-error"></div>
 <h1 th:text="${'user info for user id: ' + profileId}"></h1>
```

Comment:
Pay attention to what you're passing in from your controllers to your templates. In this case, @GetMapping("/user/{id}") adds the current logged in user as a User object, under the attribute name "user". You can access various properties of that user in Thymeleaf using "user.username", "user.firstName", etc. 

However, in this page, you're trying to access a lot of attributes that are not added in your controller. For instance, where does "profileId" come from? Is there a way you could pass that in from your controller? 

I see what you're doing with the form below, but how does Thymeleaf know what "profileUserName" means? Is there a way you could use other data you've passed in to confirm which user this is without creating a new variable?